Neos.ContentRepository.LegacyNodeMigration/Classes/Command/SiteCommandController.php

```diff
@@ -83,71 +72,84 @@ public function migrateLegacyDataCommand(bool $verbose = false, string $config =
         }
         $this->verifyDatabaseConnection($connection);
 
+        $contentRepositoryId = ContentRepositoryId::fromString($contentRepository);
+        $temporaryFilePath = $this->environment->getPathToTemporaryDirectory() . uniqid('Export', true);
+        Files::createDirectoryRecursively($temporaryFilePath);
```

Comment:
instead of operating on the real file system we could also run this in memory but that would probably overflow? (with the in memory adapter)

Reply:
php://temp can overflow into a temporary file, that could be an option